README.md

```diff
@@ -200,43 +224,105 @@ Telenor Connect has 2 [environments](http://docs.telenordigital.com/connect/envi
 that can be used, staging and production. The environment can be selected using the
 `com.telenor.connect.USE_STAGING` meta-data property in your AndroidManifest.xml
 
-```XML
+```xml
 <meta-data
         android:name="com.telenor.connect.USE_STAGING"
         android:value="true" />
 ```
 
 Set this to `false` if you want to use the production environment.
 
-### Select Client type
+### Adding the Client ID and redirect URI
+
+The Connect ID integration requires a Client ID and a redirect URI to work. You should receive these when registering your application.
+
+The Client ID and redirect URI should be added to your `strings.xml` file. Add strings with the names `connect_client_id` and `connect_redirect_uri`.
+
+```xml
+<string name="connect_client_id">example-clientid</string>
+<string name="connect_redirect_uri">example-clientid://oauth2callback</string>
+```
+
+Add `meta-data` entries to the `application` section of the manifest.
+
+```xml
+<application>
+        ...
+        <meta-data
+                android:name="com.telenor.connect.CLIENT_ID"
+                android:value="@string/connect_client_id" />
+        <meta-data
+                android:name="com.telenor.connect.REDIRECT_URI"
+                android:value="@string/connect_redirect_uri" />
+        ...
+</application>
+```
+
+### Select client type and handle the redirect URI
 
 Connect ID supports two different client types: _public_ and _confidential_. Please see the
 [Native app guide](http://docs.telenordigital.com/connect/id/native_apps.html) to help you make a
 decision.
 
-#### Confidential clients
+#### Registering the redirect URI in Android
+For your app to respond to calls to the redirect uri you need to add an `intent-filter` to your `Activity` to register this in the Android system. This will allow the [Chrome Custom Tab](https://developer.chrome.com/multidevice/android/customtabs) used by `ConnectLoginButton` and external browsers to get back to your app.
+
+```xml
+<activity android:name=".SignInActivity" >
+	<intent-filter>
+		<data android:scheme="@string/connect_client_id" />
+		<action android:name="android.intent.action.VIEW" />
+		<category android:name="android.intent.category.DEFAULT" />
+		<category android:name="android.intent.category.BROWSABLE" />
+	</intent-filter>
+</activity>
+```
+
+**Note**: If you do not wish to use the Chrome Custom Tab feature do not add this to the manifest, and the default Android WebView will be used instead.
```

Comment:
The conclusion from the team is that this branch will be initially released as a beta, to get feedback. After/when that changes I can do this.